tests/1-test-union/src/app/Client.java

```diff
@@ -29,6 +29,7 @@ private void init() {
         String ip = "localhost";
         DeviceUser duser = new DeviceUser(emsUserName, "admin", "admin");
         dev = new Device("mydev", duser, ip, 2022);
+        Element.setDebugLevel(2);
```

Comment:
magic constant?

Reply:
should not have been committed, reverting